Parser/tokenizer.c

```diff
@@ -1333,6 +1333,7 @@ verify_identifier(struct tok_state *tok)
         }
         return 0;
     }
+    assert(PyUnicode_IS_READY(s));
```

Comment:
This assert is not needed since`PyUnicode_IsIdentifier()` calls `PyUnicode_READY()`.

Reply:
True. I added this assertion mainly because `new_identifier()` in ast.c has it immediately after `PyUnicode_DecodeUTF8()`.